libr/core/cmd_debug.inc.c

```diff
@@ -3083,7 +3083,11 @@ static void cmd_debug_reg(RCore *core, const char *str) {
 					sscanf (eq, "%Lf", &val);
```

Comment:
i think that %Lf is not portable. so maybe better to use our own r_str_scanf imho